src/legacy-minter-filter-mapping.ts

```diff
@@ -337,6 +340,28 @@ export function loadOrCreateMinterFilter(
     coreContractAddress.toHexString(),
     timestamp
   );
+
+  // IMinterFilterV0 contracts may or may not have a minterFilterTypeFunction
+  // so we derive this based on the associated core contract address instead.
+  // All pre GenArt721V3 contracts should have a V0 minter filter while all
+  // GenArt721V3 contracts and above should have a V1 minter filter or higher.
+  // If the minter filter version is greater than V1 we can assume the entity
+  // creation will be handled by another event handler.
+  const coreContract = Contract.load(coreContractAddress.toHexString());
+  // We don't expect this actually happens, but we'll log a warning if it does.
+  if (!coreContract) {
+    log.warning(
```

Comment:
I've chosen not to early return here so we don't need to update anywhere that might call this function. We shouldn't ever really hit this and I believe the go wrong case is pretty harmless.